### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by abey79**
### What

`pip` seems to be sometimes worried about breaking system packages, so let's tell it to go right ahead with `--break-system-package`.

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7523?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7523?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/7523)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

**Comment by abey79** [comment]
@rerun-bot full-check

File: .github/workflows/reusable_track_size.yml [hunk 1]
```diff
@@ -192,7 +192,7 @@ jobs:
       - name: Render benchmark result
         if: github.ref == 'refs/heads/main'
         run: |
-          python3 -m pip install google-cloud-storage==2.9.0
+          python3 -m pip install --break-system-package google-cloud-storage==2.9.0
```
Comment (Wumpf) [inline .github/workflows/reusable_track_size.yml:195]: should be applied here https://github.com/rerun-io/rerun/blob/bd87d4c825922b007f0c8c9a176cc9617d40aa9f/.github/workflows/reusable_bench.yml#L161 as well

**Comment by abey79** [comment]
@rerun-bot full-check

**Comment by abey79** [comment]
@rerun-bot full-check

**Comment by abey79** [comment]
OK, full build passed 🤞🏻

**Pull request merged**